### DISCUSSION TITLE
Provide tiered feature for provisioned concurrency on lambdas (re: cold start delay)

### DISCUSSION THREAD

**Discussion by noahhai** [vercel/community#1447] [Idea]
Adding this request, originally mentioned here: https://github.com/vercel/vercel/discussions/7961#discussioncomment-4267787

Provide a pro/enterprise feature (ideally an add-on where you pass through the provisioned costs) to let users specify a Provisioned Concurrency for their functions https://aws.amazon.com/blogs/compute/operating-lambda-performance-optimization-part-1/

This would unblock customers with strict performance requirements who are experiencing issues; regardless if the issue is configuration-based with bundle-size, or underlying AWS issues, it seems many customers would be happy to defray the provisioned concurrency cost to address this issue.

**Comment by jonknyc** [comment]
We would LOVE this feature.  Cold starting our serverless functions is a huge issue for us.  We are spending a lot of effort to make services to just ping our api every so often to keep it warm - it would be so much better to just say “keep them warm” (and we’d be more than happy to pay for it).

**Comment by hiramhuang** [comment]
Yes! Take my money!
